src/commands/selection.rs

```diff
@@ -100,6 +115,81 @@ fn copy_to_clipboard(app: &mut Application) -> Result {
     Ok(())
 }
 
+/// Get the selected range from an application in a selection mode. *Requires*
+/// that the application has a buffer and is in mode Select, SelectLine, or
+/// Search.
+fn range_from(app: &mut Application) -> std::result::Result<Range, Error> {
+    let buffer = app.workspace.current_buffer();
+    let buffer = buffer.unwrap();
```

Comment:
```suggestion
    let buffer = app.workspace.current_buffer().ok_or(BUFFER_MISSING)?;
```
I know the comment mentions requiring this, but I'd rather lean on an explicit check instead of `unwrap`.

Reply:
That makes sense - thanks for noticing!